build_projects/shared-build-targets-utils/Utils/Monikers.cs

```diff
@@ -13,6 +13,7 @@ public class Monikers
         public const string CLISdkBrandName = "Microsoft .NET Core 1.0.0 - SDK Preview 2";
         public const string SharedFxBrandName = "Microsoft .NET Core 1.0.0 - Runtime";
         public const string SharedHostBrandName = "Microsoft .NET Core 1.0.0 - Host";
+        public const string HostFxrBrandName = "Microsoft .NET Core 1.0.0 - Host FX Resolver";
```

Comment:
It just seems odd to me that these strings are in CLI.  Shouldn't they be coming from the packages built in core-setup?